jwst/pipeline/calwebb_spec3.cfg

```diff
@@ -0,0 +1,18 @@
+name = "Spec3Pipeline"
+class = "jwst.pipeline.Spec3Pipeline"
+suffix = 'x1d'
+save_results = True
+
+    [steps]
+      [[skymatch]]
+        suffix = 'skymatch'
+      [[outlier_detection]]
+        suffix = 'crj'
```

Comment:
Horray!  Does that mean we can change the output product names for the image3 pipeline too from the currently-very-awkward `_cal-a3001.fits` to `_crf.fits`?